content/en/docs/Installation/raspberry.md

```diff
@@ -3,10 +3,17 @@ title: "RaspberryPi"
 linkTitle: "RaspberryPi"
 weight: 4
 date: 2022-11-13
-description: Install Kairos on RaspberryPi 3 and 4
+description: Install Kairos on RaspberryPi
 ---
 
+{{% alert title="Info" color="info" %}}
 Kairos supports Raspberry Pi model 3 and 4 with 64bit architecture.
+{{% /alert %}}
+
+{{% alert title="Warning" color="warning" %}}
+Model 5 is currently not supported because of how we use U-boot to boot the device. There's currently some work from the people from SUSE, see https://github.com/openSUSE/u-boot/pull/29 for more information.
```

Comment:
"...how **we** use..." , we do?

Reply:
I see uboot things [here](https://github.com/kairos-io/osbuilder/blob/95509370f6a87229879f1a381afa5d47225ce12d/tools-image/arm/boards/pinebookpro.sh#L12) but nothing on the rpi scripts (in the same dir). I'm not familiar with the rpi image creation but maybe we can clarify what we mean here or provide a link to the code that configures the u-boot loader for rpi.

Reply:
I mean, you say "because of how we use". Who is "we"? Kairos is not doing anything special, does it?

Reply:
@jimmykarily yes, we do something special, we use u-boot instead of putting the kernel and initramfs at a specific location like Ubuntu and Raspbian do